inventory/vagrant/group_vars/crayfish.yml

```diff
@@ -1,5 +1,7 @@
 ---
 
+crayfish_version_tag: dev-format_jsonld_strip
```

Comment:
The `dev-` here caused this to fail for me. Not sure why.

Reply:
This particular line failed for me because the ansible-role-crayfish still referenced the main Crayfish repo in the install task instead of @dannylamb's fork. Once I updated my local copy of the install task it worked fine.